### PR TITLE
Create test.yml

### PR DESCRIPTION
**Pull request by alexandria-gfsc**
Committing without secrets because to be honest I'm not 100% sure what the secrets tarball contains?

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+on: [push, pull_request, workflow_dispatch]
+
+# According to: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
+# Both Node and Yarn should be installed already
+# This would ideally be split up into dependent jobs maybe, but this is a direct conversion of the travis script for now
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```
Comment (kimadactyl) [inline .github/workflows/test.yml:11]: maybe lets set this to the same version as server

Comment (alexandria-gfsc) [reply .github/workflows/test.yml:11]: Good call, I wasn't sure if there's a way to do this! But after doing a search for other people's code it looks like there is :)

**Comment by alexandria-gfsc** [comment]
Everything works! The deploy isn't tested ATM so that needs to be done, which can only be done by merging with master :smiling_imp:

**Comment by alexandria-gfsc** [comment]
Confirmed working on kim's machine too!

**Pull request merged**